### PR TITLE
Integrate hkdf function test to strongswan

### PR DESCRIPTION
**Pull request by wangqianzou**
POO: https://progress.opensuse.org/issues/111581

- Related ticket: https://progress.opensuse.org/issues/111581
- Needles: N/A
- Verification run: 
   - Server: https://openqa.suse.de/tests/8825402

### CODE REVIEW

File: tests/fips/strongswan/strongswan_server.pm [hunk 1]
```diff
@@ -42,6 +42,13 @@ sub run {
         }
     }
 
+    # Integrate hkdf function test
+    # POO: https://progress.opensuse.org/issues/111581
```
Comment (rfan1) [inline tests/fips/strongswan/strongswan_server.pm:46]: Add the poo number to tags line as well :)

File: tests/fips/strongswan/strongswan_server.pm [hunk 2]
```diff
@@ -42,6 +42,13 @@ sub run {
...
+    assert_script_run('openssl pkeyutl -kdf HKDF -kdflen 48 -pkeyopt md:SHA256 -pkeyopt key:ff -pkeyopt salt:ff -hexdump');
+    assert_script_run('openssl pkeyutl -kdf HKDF -kdflen 48 -pkeyopt md:SHA256 -pkeyopt key:ff -pkeyopt salt:ff -pkeyopt mode:EXTRACT_ONLY -hexdump');
+    assert_script_run('openssl pkeyutl -kdf HKDF -kdflen 48 -pkeyopt md:SHA256 -pkeyopt key:ff -pkeyopt salt:ff -pkeyopt mode:EXTRACT_AND_EXPAND -hexdump');
+    assert_script_run('openssl pkeyutl -kdf HKDF -kdflen 48 -pkeyopt md:SHA256 -pkeyopt info:ff -pkeyopt key:ff -pkeyopt mode:EXPAND_ONLY -hexdump');
```
Comment (rfan1) [inline tests/fips/strongswan/strongswan_server.pm:50]: can you please also check the feature via rpm -q --changelog command to make sure the hkdf support is there?

**Review by rfan1 — APPROVED** [review]
LGTM

**Review by Amrysliu — APPROVED** [review]
LGTM

**Pull request merged**